app/Http/Controllers/SettingsController.php

```diff
@@ -2,6 +2,7 @@
 
 namespace App\Http\Controllers;
 
+use function Safe\date;
```

Comment:
See suggestion below
```suggestion
use Illuminate\Support\Carbon;
```